docs/source/jupyter/sdk.ipynb

```diff
@@ -101,18 +102,18 @@
     }
    ],
    "source": [
-    "config = {'configuration': 1}\n",
-    "transformation_id = mp.new_transformation(config)\n",
+    "from marketplace_standard_app_api.models.transformation import NewTransformationModel, TransformationId\r\n",
+    "\r\n",
+    "config = {'configuration': 1}\r\n",
+    "transformation_id: TransformationId = mp.new_transformation(NewTransformationModel(parameters=config)).id\r\n",
```

Comment:
I would suggest to get rid of the `config` variable and pass this in directly, not just for brevity, but also to avoid any confusion about a potential relation between `config` and `configuration.

However, I have not tested this, but could this potentially be shortened to:

```suggestion
    "transformation_id: TransformationId = mp.new_transformation(parameters={"some_parameter": 1}).id\r\n",
```
?